components/micropython/vfs_fs_file.c

```diff
@@ -183,9 +183,9 @@ MP_DEFINE_CONST_OBJ_TYPE(
     locals_dict, &vfs_fs_rawfile_locals_dict
 );
 
-const mp_obj_vfs_fs_file_t mp_sys_stdin_obj = {{&mp_type_vfs_fs_textio}, STDIN_FILENO};
-const mp_obj_vfs_fs_file_t mp_sys_stdout_obj = {{&mp_type_vfs_fs_textio}, STDOUT_FILENO};
-const mp_obj_vfs_fs_file_t mp_sys_stderr_obj = {{&mp_type_vfs_fs_textio}, STDERR_FILENO};
+// const mp_obj_vfs_fs_file_t mp_sys_stdin_obj = {{&mp_type_vfs_fs_textio}, STDIN_FILENO};
+// const mp_obj_vfs_fs_file_t mp_sys_stdout_obj = {{&mp_type_vfs_fs_textio}, STDOUT_FILENO};
+// const mp_obj_vfs_fs_file_t mp_sys_stderr_obj = {{&mp_type_vfs_fs_textio}, STDERR_FILENO};
```

Comment:
Won't this break Kitty on Odroid-C4?

Reply:
This doesn't break the odroidc4, this just forces micropython stdin, stdout and stderr to use serial rather than throuh the vfs. The odroidc4 doesn't require this.